test/parallel/test-http2-compat-serverresponse-end.js

```diff
@@ -25,16 +25,22 @@ const {
   // but callback will only be called once
   const server = createServer(mustCall((request, response) => {
     response.end('end', 'utf8', mustCall(() => {
-      response.end(mustNotCall());
+      response.end(mustCall());
       process.nextTick(() => {
-        response.end(mustNotCall());
+        response.end(mustCall(err => {
+          strictEqual(err.code, 'ERR_STREAM_ALREADY_FINISHED');
+        }));
         server.close();
       });
     }));
     response.on('finish', mustCall(() => {
-      response.end(mustNotCall());
+      response.end(mustCall(err => {
```

Comment:
style nit... wrap arrow function args in parens...
```suggestion
      response.end(mustCall((err) => {
```

Reply:
But better to use `common.expectError()` here instead